c-api/dotnet/src/Picky/Pem.cs

```diff
@@ -0,0 +1,170 @@
+﻿namespace Devolutions.Picky;
+
+public class PemException : System.Exception
```

Comment:
One more time: I think it's fine, but StyleCop would moan about this not being in a separate file

Reply:
*remembers bad times with Java* I'll just move in a separate file :+1:

Reply:
Oh, maybe I can define the exception class _inside_ the `Pem` class? Would that be idiomatic enough?

Reply:
Nested public classes tends to be a bit of a code smell, although there's nothing *really* wrong with it.

I'd leave it scoped the same, and move to a separate file or just leave as-is.